lib/elixir_google_scraper_web/controllers/api/v1/token_controller.ex

```diff
@@ -0,0 +1,20 @@
+defmodule ElixirGoogleScraperWeb.Api.V1.TokenController do
+  use ElixirGoogleScraperWeb, :controller
+
+  alias ElixirGoogleScraperWeb.V1.ErrorView
+  alias ElixirGoogleScraperWeb.V1.TokenView
+
+  def create(conn, params) do
+    case ExOauth2Provider.Token.grant(params, otp_app: :elixir_google_scraper) do
+      {:ok, access_token} ->
+        render(conn, TokenView, "show.json", %{
+          data: Map.put(access_token, :id, :rand.uniform(100))
```

Comment:
`ex_oauth2_provider` does not return `id` here, but `jsonapi` expects and `id` as per JSON API standard, so I am passing a random ID here

Reply:
Maybe instead of random ID, we can use something like timestamp so, it is unique for every tokens. 🤔

Reply:
It's a good idea to use timestamp instead of using rendom`ID` used `:os.system_time(:seconds)`, thanks